src/quo/previews/reacts.cljs

```diff
@@ -0,0 +1,71 @@
+(ns quo.previews.reacts
```

Comment:
Hi, Please use quo2 previews for redesign components

Reply:
Okay